maven/aem-global-parent/pom.xml

```diff
@@ -54,9 +54,17 @@
     <sling.url>http://localhost:4502</sling.url>
     <sling.user>admin</sling.user>
     <sling.password>admin</sling.password>
+    <sling.console.user>${sling.user}</sling.console.user>
+    <sling.console.password>${sling.password}</sling.console.password>
+    <sling.pkgmgr.user>${sling.user}</sling.pkgmgr.user>
```

Comment:
Technically, it's the AEM package manager, not the Sling package manager, but imho that's a minor thing.